sphinx/io.py

```diff
@@ -316,7 +316,8 @@ def read_doc(app, env, filename):
         pub = Publisher(reader=reader,
                         parser=parser,
                         writer=SphinxDummyWriter(),
-                        source_class=SphinxFileInput)
+                        source_class=SphinxFileInput,
+                        destination=NullOutput())
```

Comment:
fix setup.py build_sphinx error on Windows.
https://ci.appveyor.com/project/sphinxdoc/sphinx/builds/21581633/job/lgs76s5ekk85ygmd#L2338

Reply:
OMG... It's my wrong... I'm sorry and very thanks!